index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Dammy's Portfolio</title>
+ <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+ <!-- Header Section -->
+
+ <header>
+  <div id="page_logo">
+   <span>Dammy Shittu</span>
+  </div>
+
+  <nav>
+   <img src="./img/union.png" alt="Hamburger Image">
+  </nav>
+
+ </header>
+
+ <!-- End of Header Section -->
+
+ <!-- Main Section -->
+ <main >
+  <section class="container">
+   <div id="circles">
+    <img src="./img/ellipse-1.svg" class="circle-1" alt="Circle 1">
+    <img src="./img/ellipse-2.svg" class="circle-2" alt="Circle 2">
+   </div>
+  
+   <div id="portfolio-intro">
+    <h1>Adedamola Shittu</h1>
+    <p>Software Developer</p>
+   </div>
+ 
+   <article class="about">
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+
+    <section class="talent">
+     <div class="can-do language">
+      <h2>Language</h2>
+     </div>
+     <div class="can-do framework">
+      <h2>Framework</h2>
+     </div>
+     <div class="can-do skills">
+      <h2>Skills</h2>
+     </div>
```

Comment:
- [ ] Please try to add `<ul>` and  `<li>` tag to group the list. And use the `<a>` tag for each icon.